src/ESP32PWM.h

```diff
@@ -50,9 +57,9 @@ class ESP32PWM {
 		return (x - in_min) * (out_max - out_min) / (in_max - in_min) + out_min;
 	}
 
-	double setup(double freq, uint8_t resolution_bits=10);
+	bool setup(uint8_t pin, double freq, uint8_t resolution_bits=10);
```

Comment:
the pin is specified in the attach method not in the setup method. Please overload methods to add additions to the API and do not change existing API's. This library is used by thousands of other people, you can not change published API's.

Reply:
Thanks for the library and for reviewing this PR, and sorry for my quick hack!
I commited the changes now.